addons/binding/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/internal/ZWayBindingConstants.java

```diff
@@ -35,11 +36,12 @@
     public static final ThingTypeUID THING_TYPE_DEVICE = new ThingTypeUID(BINDING_ID, "zwayDevice");
     public static final ThingTypeUID THING_TYPE_VIRTUAL_DEVICE = new ThingTypeUID(BINDING_ID, "zwayVirtualDevice");
 
-    public static final Set<ThingTypeUID> SUPPORTED_DEVICE_THING_TYPES_UIDS = ImmutableSet.of(THING_TYPE_DEVICE,
-            THING_TYPE_VIRTUAL_DEVICE);
+    public static final Set<ThingTypeUID> SUPPORTED_DEVICE_THING_TYPES_UIDS = Collections.unmodifiableSet(
+            Arrays.asList(THING_TYPE_DEVICE, THING_TYPE_VIRTUAL_DEVICE).stream().collect(Collectors.toSet()));
```

Comment:
It's nicer/shorter to use:

`Stream.of(THING_TYPE_DEVICE, THING_TYPE_VIRTUAL_DEVICE)`

instead of:

`Arrays.asList(THING_TYPE_DEVICE, THING_TYPE_VIRTUAL_DEVICE).stream()`